### DISCUSSION TITLE
Presentation topics winter term 2024/25

### DISCUSSION THREAD

**Discussion by pkeilbach** [pkeilbach/htwg-practical-nlp#202] [Presentations]
Here is the condensed overview of presentation topics for the winter term 2024/25:

- Langchain 👉🏼 @Alex-J-W ✅ 
- Grammarly 👉🏼 @Leamayf ✅ 
- Scikit learn for NLP 👉🏼 @TobiTgl ✅ 
- PyTorch for NLP 👉🏼 @KrasserKai ✅ 
- Open Source LLMs 👉🏼 @TimSchober ✅ 
- AI Made in Germany and the EU 👉🏼 @SponsoredByPuma ✅ 
- Elasticsearch and Information Retrieval 👉🏼 @khadidja-kebaili ✅ 
- Data Storytelling 👉🏼 @mayah1010 ✅ 
- Retrieval augmented generation (RAG) 👉🏼 @ari21itwm ✅ 
- Hallucination 👉🏼 @lauraschrwr ✅ 
- [To CoT or Not to CoT? Chain of Thought helps mainly on Math and Symbolic Reasoning](https://arxiv.org/pdf/2409.12183v1) 👉🏼 @BeneHTWG ✅ 
- [Thinking LLMs: General Instruction Following with Thought Generation](https://arxiv.org/pdf/2410.10630) 👉🏼 @LilianCharlotte ✅ 
- [From Single to Multi: How LLMs Hallucinate in Multi-Document Summarization](https://arxiv.org/pdf/2410.13961) 👉🏼 @julss97 ✅

**Comment by Alex-J-W** [comment]
I added my code example from the presentation here:  [nlp_rag](https://github.com/Alex-J-W/nlp_rag). Sorry for being late
